L1Trigger/ME0Trigger/python/me0TriggerPseudoDigis_cff.py

```diff
@@ -0,0 +1,25 @@
+import FWCore.ParameterSet.Config as cms
+
+## configuration to build fast L1 ME0 trigger stubs
+## pseudo pads are created from pseudo digis with 192 strips instead of 384
+## the rechits are a necessary intermediate step before the pseudo pads are used
+## as input to build pseudo stubs
+
+from SimMuon.GEMDigitizer.muonME0PseudoReDigis_cfi import *
+from RecoLocalMuon.GEMRecHit.me0RecHits_cfi import *
+from RecoLocalMuon.GEMSegment.me0Segments_cfi import *
+
+simMuonME0PseudoReDigis192 = simMuonME0PseudoReDigis.clone(
+    numberOfStrips = cms.uint32(192)
+)
+me0RecHits192 = me0RecHits.clone(
+    me0DigiLabel = cms.InputTag("simMuonME0PseudoReDigis192")
+)
+me0TriggerPseudoDigis = me0Segments.clone(
+    me0RecHitLabel = cms.InputTag("me0RecHits192")
```

Comment:
If you change the strip granularity, you have to change the strip matching window in the segment reconstruction...see:
https://github.com/cms-sw/cmssw/blob/35ad0e0d8f48fadc1782b97e3276f1c36f5feafe/SimMuon/GEMDigitizer/python/muonME0PseudoReDigis_cfi.py#L6
https://github.com/cms-sw/cmssw/blob/35ad0e0d8f48fadc1782b97e3276f1c36f5feafe/RecoLocalMuon/GEMSegment/python/ME0SegmentsRU_cfi.py#L10